presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestingOracleServer.java

```diff
@@ -40,7 +40,7 @@ public TestingOracleServer()
         // to fix this we have to change the number of processes of SPFILE
         // but this command needs a database restart and if we restart the docker the configuration is lost.
         // configuration added:
-        // ALTER SYSTEM SET processes=500 SCOPE=SPFILE
+        // ALTER SYSTEM SET processes=1000 SCOPE=SPFILE
         // ALTER SYSTEM SET disk_asynch_io = FALSE SCOPE = SPFILE
         this.addFileSystemBind("src/test/resources/spfileXE.ora",
                 "/u01/app/oracle/product/11.2.0/xe/dbs/spfileXE.ora",
```

Comment:
if i would like to amend, or recreate the `spfileXE.ora`, how would i do that?

Reply:
Steps to update the file are:
1. Replace `BindMode.READ_ONLY` with `BindMode.READ_WRITE` at L47
2. Remove or comment out L52 ~ 55 and add
```java
            statement.execute("ALTER SYSTEM SET processes=1000 SCOPE=SPFILE");
            statement.execute("ALTER SYSTEM SET disk_asynch_io = FALSE SCOPE = SPFILE");
```
3. Run OracleQueryRunner.main

Reply:
Let me change to the above approach (restarting Oracle server inside the container).

Reply:
> Steps to update the file are:
> 
> 1. Replace `BindMode.READ_ONLY` with `BindMode.READ_WRITE` at L47
> 2. Remove or comment out L52 ~ 55 and add
> 
> ```java
>             statement.execute("ALTER SYSTEM SET processes=1000 SCOPE=SPFILE");
>             statement.execute("ALTER SYSTEM SET disk_asynch_io = FALSE SCOPE = SPFILE");
> ```
> 
> 1. Run OracleQueryRunner.main

Thanks @ebyhr!
is it documented somewhere?